harvester/harvester-installer
# feat: bump rancher to v2.9.3 and RKE2 to v1.30.7+rke2r1

Bump rancher to v2.9.3 and RKE2 to v1.30.7+rke2r1.

**Related Issue:**
https://github.com/harvester/harvester/issues/7185, https://github.com/harvester/harvester/issues/7157

**Test plan:**
1. Create a 3-node airgapped cluster without error.
2. Change "fleet-default/rke2-machineconfig-cleanup-cronjob" cronjob schedule from "5 0 * * *" to "* * * * *". Wait and check "fleet-default/rke2-machineconfig-cleanup-cronjob" job can succeed.
